specs/wp-post-editor-spec.js

```diff
@@ -892,4 +892,52 @@ test.describe( 'Editor: Posts (' + screenSize + ')', function() {
 			} );
 		} );
 	} );
+	
+	test.describe( 'Revert a post to draft', function() {
```

Comment:
Add an `@parallel` tag onto this block so that the magellan test runner can pick it up.  This is new as of a couple hours ago 😄  #504